pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1072,6 +1072,19 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 					})
 				}
```

Comment:
@davidvossel , I added a new validation webhook for the new field that we added. Thanks!